src/QueueBatch/QueueBatchTriggerAttribute.cs

```diff
@@ -47,5 +47,10 @@ public QueueBatchTriggerAttribute(string queueName)
         /// By default, the triggered function is executed only for non-empty batches. Setting this to true, will pass also the empty batches to execution.
         /// </summary>
         public bool RunWithEmptyBatch { get; set; }
+
+        /// <summary>
+        /// Gets or sets the app setting name that contains the Azure Storage connection string.
+        /// </summary>
+        public string Connection { get; set; }
```

Comment:
Would it be possible to document it in the Readme.md somehow? So far all the properties have their own sections.

Reply:
Resolved in the next commit.